### PR TITLE
LibGUI: Show minutes or hours in "unsaved changes" dialogs

### PR DESCRIPTION
**Pull request by Sauler**

### CODE REVIEW

**Comment by mustafaquraish** [comment]
I like the general idea of this, but I don't like that it does not really differentiate between say 1 hour and 1 hour and 59 minutes. It would be nice to show hours + minutes as well. More generally perhaps something that can return human-formatted time can be put into `AK::Time`?

**Comment by trflynn89** [comment]
> More generally perhaps something that can return human-formatted time can be put into `AK::Time`?

There's also `AK/NumberFormat.h` that has `human_readable_*` methods, could add `human_readable_time` there.

**Comment by Sauler** [comment]
> I like the general idea of this, but I don't like that it does not really differentiate between say 1 hour and 1 hour and 59 minutes. It would be nice to show hours + minutes as well.

I think i can easily fix that.

But method to return human readable string from time is probably topic for another PR.

File: AK/NumberFormat.h [hunk 1]
```diff
@@ -42,7 +42,29 @@ static inline String human_readable_size_long(u64 size)
         return String::formatted("{} ({} bytes)", human_readable_size(size), size);
 }
 
+static inline String human_readable_time(long long time_in_seconds)
```
Comment (trflynn89) [inline AK/NumberFormat.h:45]: We generally stray away from types like `long long`, please use `i64` instead (as that's what `AK::Time::to_seconds` returns), or templatize this function.

**Comment by Sauler** [comment]
Is there any other change required?

**Comment by trflynn89** [comment]
You'll want to post this PR in the #code-review channel on Discord to get a maintainers attention

**Comment by bgianfo** [comment]
Looks good, thanks for the PR!🎉

**Pull request merged**